src/webrtc/call.ts

```diff
@@ -700,10 +703,14 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
             purpose,
         }));
 
+        /*
+        Another commented block for retrieving existing transceivers: re-enable once we can use
```

Comment:
I don't think we should leave in comments like this we can always use git to get the code back